runatlantis.io/docs/using-atlantis.md

```diff
@@ -199,6 +199,32 @@ The automatic `env/{workspace}.tfvars` file inclusion happens during the `atlant
 
 ---
 
+## Atlantis cancel
+
+```bash
+atlantis cancel
+```
+
+### Explanation
+
+Cancels all **queued commands** for the current pull request.
+
+::: warning NOTE
+This command **does not** stop or interrupt commands that are already running. It only removes subsequent commands that are waiting in the queue. The currently running command will continue to completion.
```

Comment:
```suggestion
This command **does not** attempt to stop or interrupt commands that are already running. It only removes subsequent commands that are waiting in the queue. There is currently no mechanism in Atlantis to interrupt the currently running process.
```

A few small changes:
- Added the word "attempt" to make it clear that we don't even try
- Add a sentence to clarify that it's not a bug we don't interrupt commands, that we intentionally do not implement it at this time. Potentially that might change, but any implementation should engage with the comments in this and other threads
- Remove the third sentence (my fault, since my additional sentence made it a bit too wordy)